src/main/java/clare/Task/Deadline.java

```diff
@@ -0,0 +1,83 @@
+package clare.task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Objects;
+
+import clare.exception.StringConvertExceptions;
+
+/**
+ * Represents a task type with a deadline
+ */
+public class Deadline extends Todo {
+    private final LocalDate deadlineDate;
+    private LocalTime deadlineTime;
+
+    /**
+     * Constructor for Deadline class
```

Comment:
```suggestion
     /**
     * Constructor for Deadline class
     * @param title the task title
     * @param deadline the deadline of the task
     * @param isDone the status
     * @throws StringConvertExceptions if the format is invalid
     */
```
Should there be a blank line between the description and parameter section?

Reply:
Good recommendation, thank you!